### PR TITLE
fix create app config rollup plugin bug

### PR DESCRIPTION
**Pull request by Darmody**

### CODE REVIEW

**Comment by Darmody** [comment]
@yesmeck it's ready

File: packages/remax-cli/src/build/plugins/app.ts [hunk 1]
```diff
@@ -10,6 +22,7 @@ export default () => ({
         const declaration = path.node.declaration;
         path.replaceWith(
           t.variableDeclaration('const', [
+<<<<<<< HEAD
```
Comment (yutingzhao1991) [inline packages/remax-cli/src/build/plugins/app.ts]: 这里是有冲突吧？

Comment (yesmeck) [reply packages/remax-cli/src/build/plugins/app.ts]: @Darmody

Comment (Darmody) [reply packages/remax-cli/src/build/plugins/app.ts]: 😱 😱 😱 😱 😱

**Pull request merged**